.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Could remove this `.idea/` folder that was auto generated by GoLand? Typically, we try to not commit IDE specific files.